charts/minecraft/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: minecraft
-version: 4.20.0
+version: 4.20.1
```

Comment:
Oops, that's actually the "patch" version. Can you bump it to "4.21.0"?

Reply:
Sure can, sorry about that!